migrations/sql-migrations/dynamoToSql/Helper.js

```diff
@@ -82,12 +94,13 @@ class Helper {
       created_at: sample.createdDate,
       updated_at: sample.lastModified,
     };
-    
+
     return await this.sqlInsert(sqlSample, 'sample');
   };
-  
+
   sqlInsertSampleFile = async (sampleFileUuid, projectUuid, sample, fileName, file) => {
-    const sampleFileTypeEnumKey = this.sampleFileTypeDynamoToEnum[fileName];
+
+    const sampleFileTypeEnumKey = this.sampleFileTypeDynamoToEnum[fileName] || 'nil';
```

Comment:
`'nil'` isn't one of the possible file types declared in the enum in sql, what does having this `|| 'nil'` do to help?